kie-server-parent/kie-server-tests/kie-server-integ-tests-case-id-generator/pom.xml

```diff
@@ -99,7 +99,7 @@
               <phase>pre-integration-test</phase>
               <configuration>
                 <target>
-                  <unzip src="${maven.dependency.org.kie.server.kie-server.ee7.war.path}" dest="${project.build.directory}/unzipped-kie-server"/>
+                  <unzip src="${kie-server.war.path}" dest="${project.build.directory}/unzipped-kie-server"/>
```

Comment:
Unfortunately this doesn't work in case you run just `mvn clean install`. That could be fixed by moving `<phase>pre-integration-test</phase>` to specific profiles to this execution is skipped by default.

Reply:
@sutaakar added a skip property into the general configuration (default set to false) and per profile, overriden to true. This way, we can make both compatible, _mvn clean install_ with and without _-Pprofile_.
Wdyt?

Reply:
Sounds good to me.